src/middlewared/middlewared/plugins/service.py

```diff
@@ -607,10 +607,13 @@ def _started_notify(self, verb, what):
     async def _started_activedirectory(self, **kwargs):
         for srv in ('kinit', 'activedirectory', ):
             if await self._system('/usr/sbin/service ix-%s status' % (srv, )) != 0:
+                self.logger.debug('AD monitor: Failed to get ix-%s status' % srv)
```

Comment:
Sorry, forgot that in middlewared we are trying to get away from old '%' notation. It would be more correct now to write:
```
self.logger.debug(f'AD monitor: Failed to get ix-{srv} status')
```